cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,38 @@
 
 describe('Book Store app', () => {
   before(() => {
+    cy.visit('/login');
+  });
+
+  const username = 'ryxocus5545';
+  const password = 'Passtest!45';
+
+  it('should provide ability to log in', () => {
+    cy.get('#userName').type(username);
+
+    cy.get('#password').type(password);
+
+    cy.get('#login').click();
 
+    cy.get('#userName-value').should('contain.text', username);
+
+    cy.url().should('include', 'https://demoqa.com/profile');
   });
 
-  it('', () => {
+  it('should provide ability to search', () => {
+    cy.login(username, password);
+
+    cy.visit('/books');
+
+    cy.get('#searchBox').type('Speaking JavaScript{enter}');
```

Comment:
```suggestion
    cy.get('#searchBox').type('Speaking JavaScript{enter}');
```
create a constant for 'Speaking JavaScript' and reuse it